FWCore/Framework/bin/cmsRun.cpp

```diff
@@ -135,23 +134,10 @@ int main(int argc, char* argv[]) {
       //    be run below and can lead to error messages which should be sent via
       //    the message logger)
       context = "Initializing either multi-threaded or single-threaded message logger";
-      bool multiThreadML = false;
```

Comment:
On previous line, does the context string need to be edited to remove multi-threaded?

Reply:
Good point